src/main/resources/SoapExceptionMessages.properties

```diff
@@ -2,3 +2,4 @@ E_SERVICE_AUTHORIZATION_FAILED_INVALID_IDENTITIES=Authorization failed, please r
 E_SERVICE_AUTHORIZATION_FAILED=Authorization failed, please review the identities provided
 E_UNKNOWN_TOKEN=Token "{0}" not found in registry central. Please make sure you are using the right registry and the token is not expired.
 E_UNSUPPORTED_EXCEPTION=An unexpected service exception occurred please check logs
+E_BASIC_AUTHORISATION_FAILED=request from service "{0}" not allowed with basic authorisation
```

Comment:
We cant introduce a new Error code as these error code were taken from Legacy system and client can only interpret it. Please use E_SERVICE_AUTHORIZATION_FAILED.